### PR TITLE
Fixes from menkalinan

### PR DESCRIPTION
**Pull request by justinjmoses**
* Owner script fixes
* Adding L2 relayer as owner

### CODE REVIEW

File: publish/src/commands/owner.js [hunk 1]
```diff
@@ -379,14 +380,14 @@ const owner = async ({
 		const batchSize = 20;
 
 		for (let i = 0; i < actions.length; i += batchSize) {
-			const batchActions = actions.slice(i, batchSize);
+			const batchActions = actions.slice(i, i + batchSize);
```
Comment (justinjmoses) [inline publish/src/commands/owner.js:383]: ouch

File: publish/src/commands/owner.js [hunk 2]
```diff
@@ -379,14 +380,14 @@ const owner = async ({
...
 			const batchData = OwnerRelayOnEthereum.interface.encodeFunctionData('initiateRelayBatch', [
 				batchActions.map(({ target }) => target),
 				batchActions.map(({ data }) => data),
 				ethers.BigNumber.from('0'),
 			]);
 			if (safeBatchSubmitter) {
-				safeBatchSubmitter.appendTransaction({
+				await safeBatchSubmitter.appendTransaction({
```
Comment (justinjmoses) [inline publish/src/commands/owner.js:390]: double ouch

File: index.js [hunk 3]
```diff
@@ -492,7 +492,7 @@ const getUsers = ({ network = 'mainnet', user, useOvm = false } = {}) => {
 		kovan: Object.assign({}, base),
 		'kovan-ovm': Object.assign({}, base),
```
Comment (barrasso) [inline index.js:493]: [np] did we want to add the `kovan-ovm` owner here?

File: publish/src/commands/owner.js (hunk 1 above)
Comment (barrasso) [reply publish/src/commands/owner.js:383]: :comeonmate:

**Review by barrasso — APPROVED** [review]
LGTM, just one minor question on the `index.js`

File: publish/src/commands/owner.js (hunk 2 above)
Comment (barrasso) [reply publish/src/commands/owner.js:390]: :comeonemate:^2

**Pull request merged**